setup.js

```diff
@@ -73,7 +73,7 @@ setup.InsertTemplate = function(req, templateObj){
     // set the required authentication information
     let dsApiClient = new docusign.ApiClient();
     dsApiClient.setBasePath(req.session.basePath);
-    dsApiClient.addDefaultHeader('Authorization', 'Bearer ' + req.session.access_token);
+    dsApiClient.addDefaultHeader('Authorization', 'Bearer ' + dsAuthCodeGrant.prototype.getAccessToken());
```

Comment:
Do we use auth code grant in LoanCo?? I thought it's JWT only, if it's using JWT but the name is authCodeGrant for the variable, can we fix? it may confused developers...